contributors.json

```diff
@@ -52,5 +52,9 @@
   "WickedInvi": {
     "country": "United Kingdom",
     "name": "Deyan Petrov"
+  },
+  "Pharcside": {
```

Comment:
Hi @Pharcside. 
Thanks for your contribution. In order to avoid merge conflicts, currently, we need to order the names alphabetical case insensitive in the JSON file. You can manually order this or if you have some knowledge of node js you can do this automatically by using the following command:
```
npx sort-json contributors.json --ignore-case
```

If you have difficulties in the ordering of objects I'll be glad to help :)

Reply:
Ill try and fix, thanks!